README.md

```diff
@@ -1,27 +1,11 @@
 [ ![Codeship Status for databraid-dashboard/transit-api](https://app.codeship.com/projects/459d8c80-5ead-0135-6b43-16bdadfc7181/status?branch=master)](https://app.codeship.com/projects/238758)
 
-# Databraid API Starter Repo Without Postgres
+# Databraid Transit API
 
-Use this repo as a starter repo for developing API servers without Postgres
+## Description
 
-## Usage
-
-Before beginning make sure you have Docker installed and running, then do:
-
-```
-npm i
-npm run up
-```
+An API that uses Express to give realtime public transit directions from a starting point to a destination and public transit alerts for the San Francisco Bay Area. This API uses the Google Directions API to retrieve multiple routes and travel information between an origin and a destination for respective routes by public transit. This API uses uses the 511.org API to retrieve alerts information for the bay area and parse the alerts for information on which bus lines are affected.
 
-When finished developing do:
-
-```
-npm run down
-```
-
-Testing and linting can all be done respectively with:
+## Usage
 
-```
-npm test
-npm run lint
-```
+Before beginning, make a .env file containing a Google Maps API key for the GOOGLE_MAPS_API_KEY variable and a 511.org API key for the FIVE_ONE_ONE_API_KEY variable. Also set what port the API is running on with the PORT variable.
```

Comment:
Please wrap the filename for .env in back ticks so that is is fixed width: `.env`

Reply:
And please also do the same for each of the environmental variables.